package.json

```diff
@@ -8,6 +8,7 @@
     "check-types": "tsc",
     "preview": "vite preview",
     "lint": "eslint --max-warnings 0 'src/**/*.ts' 'src/**/*.tsx' && knip",
+    "lighthouse": "SCRIVITO_TENANT=d0a154d76edf2a7bd991fc658e700a1d SCRIVITO_ORIGIN=http://localhost:8080 PRERENDER_OBJ_ID=c2a0aab78be05a4e npm run prerender && precompress --include '(**).(css|html|js)' --types br dist-ssr && for run in $(seq 6); do lhci autorun && break; done",
```

Comment:
Regarding `precompress --include '(**).(css|html|js)' --types br dist-ssr` - similar to the `startServerCommand` comment from above: Could this be consolidated with `npm run prerender-preview`?